<?xml version="1.0" encoding="UTF-8"?>
<alr:test-suite xmlns:alr="urn:model.allure.qatools.yandex.ru" start="1556668853422" stop="1556668853425" version="1.4.0">
  <name>Magento\Framework\Exception\Test\Unit\InputExceptionTest</name>
  <test-cases>
    <test-case start="1556668853422" stop="1556668853423" status="passed">
      <name>testConstructor</name>
    </test-case>
    <test-case start="1556668853423" stop="1556668853424" status="passed">
      <name>testAddError</name>
    </test-case>
    <test-case start="1556668853424" stop="1556668853425" status="passed">
      <name>testAddErrorWithSameMessage</name>
    </test-case>
  </test-cases>
</alr:test-suite>
